Coordinates pos
---
float64 rot
